### DISCUSSION TITLE
Welcome to Shareish Discussions!

### DISCUSSION THREAD

**Discussion by rmaree** [shareish/shareish#74] [General]
## 👋 Welcome to Shareish Discussions !
  We’re using GitHub Discussions as a place to connect with other members of our community, namely Shareish users, developers, and IT administrators. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

 If you are looking for how to contribute, please have a look to our [CONTRIBUTING](https://github.com/shareish/shareish/blob/main/CONTRIBUTING.md) page !

**Comment by MaelDre** [comment]
Hi There, just a short message to congratulate about the work done ! It's an amazing project :+1: 
And to introduce me shortly I've been interesting myself for a while about sharing / mutualizing stuffs, and I would like to build a community of sharing or a Library of things in the place where I live now (in Brittany west of France).
I'm afraid not to be skilled enough in vue.js/Python to help you, but we'll see :-)

**Reply by rmaree** [reply]
Hello Mael, thanks for your enthusiast message ! Feel free to try/install Shareish and give us your feedback (missing features, suggestions, bug reports, translation issues, ...), it's another way to help/contribute ! (we also speak french)
